### PR TITLE
Fixing live tests

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

**Comment by seanmcc-msft** [comment]
@kasobol-msft @amnguye

File: sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs [hunk 1]
```diff
@@ -403,7 +419,25 @@ public async Task CreateAsync_WithBlobServiceSas()
...
+                containerName: containerName,
+                permissions: permissions,
+                sasVersion: ToSasVersion(_serviceVersion));
+
+            BlobUriBuilder blobUriBuilder = new BlobUriBuilder(service.GetBlobContainerClient(containerName).Uri)
+            {
+                Sas = sasQueryParameters
+            };
+
+            BlobContainerClient container = new BlobContainerClient(blobUriBuilder.ToUri(), GetOptions());
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs:440]: Nit - we need to instrument the client

File: sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs [hunk 2]
```diff
@@ -573,6 +573,7 @@ public async Task Batch_Double_Dispose_Response_Still_Available()
         }
 
         [Test]
+        [Ignore("service bug")]
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs:576]: Nit - let's file a bug in our repo to re-enable this

File: sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs [hunk 3]
```diff
@@ -595,6 +596,7 @@ public async Task SetBlobAccessTier_Snapshot()
...
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs:599]: Nit - let's file a bug in our repo to re-enable this

File: sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs [hunk 4]
```diff
@@ -2685,11 +2685,17 @@ public async Task GetPropertiesAsync_ContainerSAS()
...
+                containerName: test.Container.Name,
+                permissions: BlobContainerSasPermissions.Read,
+                sasVersion: ToSasVersion(_serviceVersion));
+
+            BlobUriBuilder blobUriBuilder = new BlobUriBuilder(blob.Uri)
+            {
+                Sas = blobSasQueryParameters
+            };
+
+            BlockBlobClient sasBlob = new BlockBlobClient(blobUriBuilder.ToUri(), GetOptions());
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs:2698]: Nit - need to instrument client

File: sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs [hunk 5]
```diff
@@ -2718,12 +2724,19 @@ public async Task GetPropertiesAsync_ContainerIdentitySAS()
...
+                userDelegationKey: userDelegationKey,
+                TestConfigOAuth.AccountName,
+                sasVersion: ToSasVersion(_serviceVersion));
+
+            BlobUriBuilder blobUriBuilder = new BlobUriBuilder(blob.Uri)
+            {
+                Sas = blobSasQueryParameters
+            };
+
+            BlockBlobClient identitySasBlob = new BlockBlobClient(blobUriBuilder.ToUri(), GetOptions());
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs:2739]: Nit - need to instrument client

File: sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs [hunk 6]
```diff
@@ -2995,13 +3008,21 @@ public async Task GetPropertiesAsync_SnapshotIdentitySAS()
...
+                userDelegationKey: userDelegationKey,
+                accountName: TestConfigOAuth.AccountName,
+                sasVersion: ToSasVersion(_serviceVersion));
+
+            BlobUriBuilder blobUriBuilder = new BlobUriBuilder(blob.Uri)
+            {
+                Sas = blobSasQueryParameters
+            };
+
+            BlockBlobClient identitySasBlob = (new BlockBlobClient(blobUriBuilder.ToUri(), GetOptions())).WithSnapshot(snapshotResponse.Value.Snapshot);
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs:3025]: Nit - need to instrument client

**Review by tg-msft — APPROVED** [review]
Nice job getting these green!

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs:576]: https://github.com/Azure/azure-sdk-for-net/issues/13507

File: sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs (hunk 3 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs:599]: https://github.com/Azure/azure-sdk-for-net/issues/13507

File: sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs]: I'm pretty sure new XClient(..., GetOptions()) is the same as InstrumentClient(new XClient())

Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs]: No, they're different.  `GetOptions` will setup recording.  `InstrumentClient` will setup the sync/async and distributed tracing verification.

**Pull request merged**